src/Message/DirectPostCompletePurchaseRequest.php

```diff
@@ -11,9 +11,9 @@ class DirectPostCompletePurchaseRequest extends DirectPostAbstractRequest
 {
     public function getData()
     {
-        $data = $this->httpRequest->request->all();
+            $data = $this->httpRequest->query->all();
```

Comment:
I dont know why but `$this->httpRequest->request->all()` do not work to me, and I have doubts it ever worked